docs/source/azure/advanced/azurefunctions.rst

```diff
@@ -36,12 +36,13 @@ Execution in Azure functions comes with a default set of configurations for the
 resources. To override these settings you must set 'provision-options' with one of the following
 keys:
 
+- groupName (default: cloud-custodian)
```

Comment:
It is not clear to me until I look at the code that this groupName also has to be the groupName where the storage account is. We should either specify that in the name 'storageAccountResourceGroup' or allow the user to pass 'groupName' and 'storageAccountResourceGroup' if they want to be able to provision everything in a new resource group separate from their storage account one.